syncd/FlexCounter.cpp

```diff
@@ -114,6 +114,14 @@ FlexCounter::MACsecSAAttrIds::MACsecSAAttrIds(
     // empty intentionally
 }
 
+FlexCounter::TunnelCounterIds::TunnelCounterIds(
+        _In_ sai_object_id_t tunnel,
+        _In_ const std::vector<sai_tunnel_stat_t> &tunnelIds):
+    tunnelId(tunnel), tunnelCounterIds(tunnelIds)
+{
+    SWSS_LOG_ENTER();
+}
```

Comment:
add // empty
to notify that this is empty intentionally

Reply:
Done